Orange/widgets/data/owfile.py

```diff
@@ -21,128 +22,7 @@ def add_origin(examples, filename):
             var.attributes["origin"] = dir_name
 
 
-class RecentPath:
```

Comment:
Please import RecentPath item from the new location into this module's namespace so that settings saved before the change can be loaded.